src/core/sock/sockinfo_tcp.h

```diff
@@ -512,20 +512,25 @@ class sockinfo_tcp : public sockinfo, public timer_handler {
     list_node<sockinfo_tcp, sockinfo_tcp::accepted_conns_node_offset> accepted_conns_node;
 
 private:
+    lock_dummy m_tcp_con_lock;
```

Comment:
Currently we can't remove locks because some control path sockets should still be protected.